delfin/drivers/dell_emc/unity/unity.py

```diff
@@ -492,102 +492,53 @@ def list_shares(self, context):
             share_list.extend(self.get_share('nfs', qtrees))
             return share_list
         except Exception as err:
-            err_msg = "Failed to get shares metrics from Unity: %s"\
+            err_msg = "Failed to get shares attributes from Unity: %s"\
                       % (six.text_type(err))
             raise exception.InvalidResults(err_msg)
 
     def get_tree_quotas(self):
         quotas_list = []
-        quota_configs = self.rest_handler.get_quota_configs()
         qts = self.rest_handler.get_all_qtrees()
         if qts is None:
             return quotas_list
         qt_entries = qts.get('entries')
-        conf_entries = quota_configs.get('entries')
         for quota in qt_entries:
-            capacity_hard_limit = 0
-            capacity_soft_limit = 0
-            file_hard_limit = 0
-            file_soft_limit = 0
-            limit_type = 'block'
             content = quota.get('content')
             if not content:
                 continue
-            for conf in conf_entries:
-                conf_content = conf.get('content')
-                if not conf_content:
-                    continue
-                if conf_content.get('id') == content.get(
-                        'quotaConfig').get('id'):
-                    if int(conf_content.get('quotaPolicy')) == 0:
-                        limit_type = 'file'
-                    break
-            if limit_type == 'file':
-                file_hard_limit = content.get('hardLimit')
-                file_soft_limit = content.get('softLimit')
-            else:
-                capacity_hard_limit = content.get('hardLimit')
-                capacity_soft_limit = content.get('softLimit')
             qt = {
                 "native_quota_id": content.get('id'),
                 "type": constants.QuotaType.TREE,
                 "storage_id": self.storage_id,
                 "native_filesystem_id":
                     content.get('filesystem', {}).get('id'),
                 "native_qtree_id": content.get('id'),
-                "capacity_hard_limit": capacity_hard_limit,
-                "capacity_soft_limit": capacity_soft_limit,
-                "file_hard_limit": file_hard_limit,
```

Comment:
File limit not needed anymore?

Reply:
yes，and in the mananger gui，there is no file limit too